assemblies/assembly_managing-and-monitoring-business-processes/main.adoc

```diff
@@ -50,6 +49,12 @@ include::{enterprise-dir}/admin-and-config/tasks-default-filters-proc.adoc[level
 include::{enterprise-dir}/admin-and-config/viewing-task-variables-basic-filters-in-bc-proc.adoc[leveloffset=+2]
 include::{enterprise-dir}/admin-and-config/viewing-task-variables-advanced-filters-in-bc-proc.adoc[leveloffset=+2]
 include::{enterprise-dir}/admin-and-config/manage-service-tasks-proc.adoc[leveloffset=+2]
+include::{enterprise-dir}/admin-and-config/managing-business-processes-bulk-actions-on-tasks-con.adoc[leveloffset=+2]
```

Comment:
Move the section one only after "4.8. User task administration", i.e. under line
```
include::{jbpm-dir}/BPMN2/user-task-admin-con.adoc[leveloffset=+2]
```

Currently we're talking about bulk actions even before we introduce the basic operations for tasks.

Reply:
@jstastny-cz, Ack. Thanks!